docs/concepts/feathr-concepts-for-beginners.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: Feathr Concepts for Beginners
+nav_order: 2
+---
+
+# Feathr Concepts for Beginners
+
+In this guide, we will cover the high level concepts for Feathr. Don't treat this as a user manual, instead treat this as blogpost to cover the highlevel motivations on why Feathr introduces those concepts.
+
+## What are `Observation` data, and why does Feathr need `key(s)`, `Anchor`, `Source`?
+
+In order to fully utilize Feathr's power, we need to understand the object models that Feathr is expecting.
+
+In Feathr, always think that there is some `Observation` dataset which is the central dataset that you will be using. The observation data set will usually have at least two columns: a timestamp column (indicating when this observation happens) and a column containing IDs, and with other possible fields
+
+For example, the `observation` dataset can be user click streams, credit card transactions, etc., while the data science team is trying to predict whether the user clicks on something, or whether the transcation is a fraud trasction, etc.
```

Comment:
we can just explain from the objective of DS(guess you are more familiar with that).

For example, as a DS, you want to build a model to do sth, then define obs ....

Then explain typically what was done without Feathr. But with Feathr, how things can be made easier.